static/src/javascripts/projects/commercial/modules/header-bidding/a9/a9.js

```diff
@@ -9,7 +9,13 @@ import type {
     HeaderBiddingSize,
     HeaderBiddingSlot,
 } from 'commercial/modules/header-bidding/types';
-import { onIabConsentNotification } from '@guardian/consent-management-platform';
+
+import { onConsentChange, oldCmp } from '@guardian/consent-management-platform';
+import { isInTcfv2Test } from 'commercial/modules/cmp/tcfv2-test';
+
+const onIabConsentNotification = isInTcfv2Test()
```

Comment:
Did you forget to call it `onCMPConsentNotification` here like in the other places?
I think you also meant to update the callback that gets passed into `onIabConsentNotification` to be able to account for the TCFv2 state objects on line 42 of this file.